Wikipedia/Code/WMFImageGalleryDetailOverlayView.m

```diff
@@ -73,16 +36,25 @@ - (IBAction)didTapInfoButton {
     }
 }
 
+- (IBAction)didTapImageDescription {
```

Comment:
Could this be propagated up to the View Controller via a delegate so that this doesn't have to tell the superview to `layoutIfNeeded`?

Reply:
sure - i'll try to move the other `didTap` methods right above this one while i'm at it...

Reply:
fixed!